setup.py

```diff
@@ -0,0 +1,19 @@
+from setuptools import setup
+
+dependencies = ['pytest', 'pytest-cov']
+extra_packages = {
+    'testing': ['tox']
+}
+
+setup(
+    name='Linked List',
```

Comment:
Is this an appropriate name for this package, which will house multiple data structures and even a few algorithms?